index.js

```diff
@@ -37,9 +37,21 @@ let Web3;
 if (typeof window === 'undefined') {
   Web3 = require("web3"); // eslint-disable-line
 } else {
-  Web3 = window.Web3; // eslint-disable-line
+  if (!window.Web3) {
+    Web3 = require('web3')
+  }else {
```

Comment:
please use eslint for code format and linting

Reply:
👌OK, I will use eslint